cmd/Dockerfile

```diff
@@ -70,3 +70,5 @@ FROM gcr.io/distroless/base-debian11 as production-transit-nocgo
 COPY --from=build-transit-production-nocgo /app/app /bin/app
 
 ENTRYPOINT ["/bin/app"]
+
+FROM build-setup as environment-clean
```

Comment:
This thin docker image exists because I was unable to run a docker build on build-setup without major extra work, this thin image does all the same work but is available directly in the Makefile